spec/services/case_validate_rejected_offender_sar_service_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+describe CaseValidateRejectedOffenderSARService do
+  describe "#call" do
+    let(:team)          { find_or_create :team_branston }
+    let(:user)          { find_or_create :branston_user }
+    let(:kase)          { create :offender_sar_case, :rejected, received_date: 1.day.ago.to_date }
+    let(:state_machine) { double ConfigurableStateMachine::Machine,  validate_rejected_case!: true } # rubocop:disable RSpec/VerifiedDoubles
+    let(:service)       do
+      described_class.new(user:,
+                          kase:,
+                          params:)
+    end
+
+    before do
+      allow(kase).to receive(:state_machine).and_return(state_machine)
```

Comment:
I'm not sure this is necessary. Can't you check the actual state_machine received the expected method call:
```
expect(kase.state_machine).to receive(:validate_rejected_case!)
```